problem_builder/swipe.py

```diff
@@ -136,15 +116,9 @@ def student_view_data(self, context=None):
             'block_id': unicode(self.scope_ids.usage_id),
             'display_name': self.display_name_with_default,
             'type': self.CATEGORY,
-            'question': self.question,
-            'message': self.message,
+            'text': self.text,
             'img_url': self.expand_static_url(self.img_url),
-            'choices': [
-                {'value': choice['value'], 'content': choice['display_name']}
-                for choice in self.human_readable_choices
-            ],
-            'weight': self.weight,
-            'tips': [tip.student_view_data() for tip in self.get_tips()],
+            'correct': self.student_choice == self.correct,
```

Comment:
This should just be `'correct': self.correct`

(We can't access student-specific fields like `student_choice` in `student_view_data` anyways - only in `student_view_user_state`.)

Reply:
Ah, ok! :+1: